### ISSUE TITLE
save model history as file for UNET

### ISSUE THREAD

**Issue by octavifs** [octavifs/lucanode#7]
keras model.fit_generator returns a History object. This object stores the loss function and metrics for both the training and validation on each epoch. Save this output as a csv when storing the model (as .h5)

This is going to be useful to plot the evolution of the loss function over time and spot overfitting.
